src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -262,10 +271,18 @@ temperature_archery <- all_data %>%
             diff = abs(min(value, na.rm = T) - max(value, na.rm = T)),
             n_obs = n()) %>% 
   ungroup() %>%
+    arrange(DT_round) %>%
   # pad the dataset so that all 15-min timestamps are present
-  pad(by = "DT_round") %>% 
+  pad(by = "DT_round", interval = "15 min") %>%
+  mutate(DT_join = as.character(DT_round)) %>%
+  full_join(filter(dplyr::select(field_notes, where, visit_comments, sensor_malfunction, DT_join, site), site == "archery"), 
```

Comment:
question (non-blocking): this looks correct to me, is it possible to pull these merge request changes into my local repo so that I can test them on my computer before I accept those changes?

Reply:
Great question: Best way to do that (I think) is to clone Katie's repo. I have a folder in my GH folder called 'PR', where I'll clone the author's repo so I can run things on my end - and then I don't confuse that clone with my forks. For exceptional GH hygiene, you can immediately delete the remotes so that you don't accidentally push things into the OP's repo.

Reply:
Thanks B, I'll set up a PR folder as well.